docs/observability/how_to_guides/tracing/mask_inputs_outputs.mdx

```diff
@@ -292,3 +292,317 @@ async def async_function(key: str) -> int:
 ```
 
 These function-level processors take precedence over client-level processors (`hide_inputs` and `hide_outputs`) when both are defined.
+
+## Quick starts
+
+You can combine LangSmith's rule-based masking with various anonymizers to scrub sensitive information from inputs and outputs. In this how-to-guide, we'll cover working with Microsoft Presidio, AWS Comprehend, and regex.
+
+### Microsoft Presidio
+
+Microsoft Presidio is a data protection and de-identification SDK. The implementation below uses Presidio to anonymize inputs and outputs before they are sent to LangSmith. For up to date information, please refer to Presidio's [official documentation](https://microsoft.github.io/presidio/).
+
+```python
+import openai
+from langsmith import Client
+from langsmith.wrappers import wrap_openai
+from langchain_experimental.data_anonymizer import PresidioAnonymizer
+
+anonymizer = PresidioAnonymizer()
+
```

Comment:
This code gives me an error:
```
Failed to post run 9dc2bfdc-539d-4574-b2d1-1b184cd04392: AnonymizerEngine._remove_conflicts_and_get_text_manipulation_data() takes 2 positional arguments but 3 were given
Unable to process trace outputs: TypeError('AnonymizerEngine._remove_conflicts_and_get_text_manipulation_data() takes 2 positional arguments but 3 were given')
```

Reply:
strange it works for me. likely a SDK version issue. ill implement with pure presidio should solve it